helm/cluster-operator-chart/templates/configmap.yaml

```diff
@@ -8,6 +8,14 @@ data:
     server:
       listen:
         address: 'http://0.0.0.0:8000'
+    guest:
+      cluster:
+        kubernetes:
+          api:
+            clusterIPRange: '{{ .Values.Installation.V1.Guest.Kubernetes.API.ClusterIPRange }}'
+        vault:
+          certificate:
+            ttl: '{{ .Values.Installation.V1.Auth.Vault.Certificate.TTL }}'
```

Comment:
I'd like to see those with the PR they are used with (to learn how they are used).

Reply:
This PR actually uses those. This is something I had missed earlier because of partially laziness, partially misunderstanding. I counted too much on unit tests on some of recent PRs and didn't test every time on actual installation because I thought nothing changes in execution because there's nothing that would activate, but this was incorrect assumption. The initialization reads those configuration values here:
- https://github.com/giantswarm/cluster-operator/blob/master/service/service.go#L157
- https://github.com/giantswarm/cluster-operator/blob/master/service/service.go#L163

...and CIDR parsing failed because those configuration values were not present (nor actually registered in `main.go`). Now they are and this works on actual installation again. Lesson learned: *always* test on actual installation :flushed:

Reply:
Oh and those configuration values are eventually used in `certconfig` resource which uses that information for certificate creation.